src/components/LangSelector/LangSelector.tsx

```diff
@@ -0,0 +1,33 @@
+import { Select } from '@beercode/common-frontend'
+import { Locale } from 'src/types/localization'
+import { useLocale } from 'src/hooks/useLocale'
+import { useRouter } from 'next/router'
+
+export const LangSelector: React.FC = () => {
+  const router = useRouter()
+  const { changeLocale } = useLocale()
+  const options = [
```

Comment:
u should use array from types/localization
like Object.keys(Locale)

Reply:
use should create type for options like Record<string, Locale>
may be create it in LngSelector.types file and called like LngOption

Reply:
then ur options should be look like options: LngOption[]